test/tests/val-2c/val-2c_delay.params

```diff
@@ -14,8 +14,8 @@ volume_enclosure = ${units 0.96 m^3 -> mum^3}
 temperature = ${units 303 K}
 
 ## Conversion
-Curie = ${units 3.7e10 1/s} # desintegrations/s - activity of one Curie
-decay_rate_tritium = ${units 1.78199e-9 1/s/at} # desintegrations/s/atoms
+Curie = ${units 3.7e10 1/s} # disintegrations/s - activity of one Curie
+decay_rate_tritium = ${units 1.78199e-9 1/s/at} # disintegrations/s/atoms
```

Comment:
Could you go through the verification and validation cases, as well as examples, and update the decay rate used in all the relevant cases to use the same value as the new one you use. Please also make sure to update the documentation pages, when relevant.